Atala Platform/2023.11.10.md

```diff
@@ -0,0 +1,11 @@
+# Atala PRISM v2.5
+
+November 10 2023 release of the Atala Platform
+
+## Changelog
+
+**New Features:**
+
+- Environments Status Dashboard - A new dashboard showing all Atala 2.x Environments is now available at
+  https://atala.grafana.net/d/de0dedc4-0c84-4da8-980a-c69f521fd8ba/environment-status?orgId=1
+- ArgoCD Slack Notifications - Deployments status are now sent to internal slack channel #atala-release-notifications
```

Comment:
I would consider to add:
- a section that indicates what benefit the feature is adding
- a section that is asking the recipient of the release what is asked them to do with the new feature. Example on the environment dashboard, should the user click and connect to the dashboard and then what he/she can do with it? Same for the slack notification. Is there any feedback your are asking with this feature? (format of the info in the notification)
- a documentation section to this release template: if a user wants to know more about the added features, where to go